### PR TITLE
Build: abs added

### PR DESCRIPTION
**Pull request by skupr-anaconda**

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+aggregate_check: false
```
Comment (katietz) [inline abs.yaml:2]: no, please remove this aggregate_check

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,43 +1,45 @@
...
-
+  sha256: f2b065d1c732e8e1828fabe720dc82f12e6109d0d2bbfbcccc17eaefae0fd7c9
 
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<38]
+  # skipping s390x as we do not have setproctitle package for this arch. Also, the current package 
+  # will go to Snowflake-specific channel only, so there is no urgency to build it on s390x.
+  skip: true  # [s390x]
```
Comment (katietz) [inline recipe/meta.yaml:17]: please write here ... [linux and s390x] ... using s390x (or other arch names without OS) might be problematic

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,43 +1,45 @@
...
+    - wheel
+    - python 
+    - setuptools      <64
   run:
-    - click >=7.0,!=8.0.0
-    - docker-pycreds >=0.4.0
-    - gitpython >=1.0.0
+    - click           >=7.0,!=8.0.0
+    - docker-pycreds  >=0.4.0
+    - gitpython       >=1.0.0,!=3.1.29
```
Comment (katietz) [inline recipe/meta.yaml:32]: do not add multiple spaces before the version constrain.  One is enough

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,43 +1,45 @@
...
     - pathtools
-    - protobuf >=3.19.0,!=4.21.0,<5
-    - psutil >=5.0.0
-    - python >=3.6
+    - protobuf        >=3.20.3,!=4.21.0,<5
+    - psutil          >=5.0.0
```
Comment (katietz) [inline recipe/meta.yaml:35]: see above ... likewise

**Review by katietz — CHANGES_REQUESTED** [review]

**Review by varlackc — COMMENTED** [review]
There seems to be a change in the `dev_url`

from 

https://github.com/wandb/client

to

https://github.com/wandb/wandb

**Review by skupr-anaconda — COMMENTED** [review]
LGTM

File: recipe/meta.yaml (hunk 4 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Done

File: recipe/meta.yaml (hunk 3 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Done and noted

File: recipe/meta.yaml (hunk 2 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Done and noted

**Review by katietz — APPROVED** [review]
thanks

**Comment by akabanovs** [comment]
Package has been uploaded to the customer-specific channel.

**Pull request merged**